acestep/training/data_module_test.py

```
@@ -0,0 +1,121 @@
+    def test_fallback_scan_only_finds_pt_files(self):
+        """Without manifest, only .pt files inside tensor_dir are found."""
+        with tempfile.TemporaryDirectory() as d:
+            for name in ["a.pt", "b.pt", "c.txt"]:
+                open(os.path.join(d, name), "wb").close()
+
+            ds = PreprocessedTensorDataset(d)
+            self.assertEqual(len(ds.valid_paths), 2)
+
+    def test_nonexistent_dir_raises(self):
+        with self.assertRaises(ValueError):
+            PreprocessedTensorDataset("/tmp/nonexistent_xyz_12345")
```

Ruff (0.15.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent_xyz_12345"

(S108)